plumgrid_playbooks/roles/plumgrid_common/tasks/main.yml

```diff
@@ -45,51 +40,37 @@
     file: /tmp/GPG-KEY
     state: present
 
-# Update repositories
-- name: Running apt-update
```

Comment:
I'm wondering if this behaviour might be consistent between different ansible versions. I prefer the idea of a common apt update at the start.

Also the problem in this task seems to be that you missed adding {{ item }} infront of name